Projects/UOContent/Spells/Base/MagerySpell.cs

```diff
@@ -5,49 +5,44 @@ namespace Server.Spells
 {
     public abstract class MagerySpell : Spell
     {
-        private static readonly int[] _manaTable = { 4, 6, 9, 11, 14, 20, 40, 50 };
-
-        /*
-         * Starts at Circle -2 to account for scrolls
-         * Mana requirements formula: (14 * (circle - 1)) + 2 = 50% probability
-         * Add or subtract 20 for max or min limits
-         */
-        private static readonly double[] _requiredSkill = Core.ML ?
-            new[] { -46.0, -32.0, -18.0, -4.0, 10.0, 24.0, 38.0, 52.0, 66.0, 80.0 } :
-            new[] { -50.0, -30.0, 0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0 };
-
-        public MagerySpell(Mobile caster, Item scroll, SpellInfo info) : base(caster, scroll, info)
-        {
-        }
+        // Mana costs per spell circle
+        public static int[] ManaPerCircle { get; set; } = { 4, 6, 9, 11, 14, 20, 40, 50 };
+
+        // Minimum skill required per circle (scrolls use Circle+2) for non-ML core (10 entries)
+        public static double[] SkillTable { get; set; } = { -50.0, -30.0, 0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0 };
+        public static double[] SkillTableML { get; set; } = { 0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0 };
```

Comment:
I think this got changed from the values I had. Do we have info on these values and if they were wrong before, or they are correct with these values?